src/lib/markdown/operationLink.ts

```
@@ -0,0 +1,171 @@
+  } = options
+
+  const openapi = useOpenapi()
+
+  // Override the link renderer.
+  md.renderer.rules.link_open = function (tokens, idx, options, env, self) {
+    const hrefIndex = tokens[idx].attrIndex('href')
+    if (hrefIndex < 0) {
+      return defaultRender(tokens, idx, options, env, self)
+    }
+
+    const href = tokens[idx].attrs![hrefIndex][1]
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)